assets/js/autocomplete.ts

```diff
@@ -105,19 +107,44 @@ function keydownHandler(event: KeyboardEvent) {
   }
 }
 
-function findSelectedTerm(targetInput: HTMLInputElement, searchQuery: string): TermContext | null {
+function findSelectedTerm(targetInput: InputFieldElement, searchQuery: string): TermContext | null {
   if (targetInput.selectionStart === null || targetInput.selectionEnd === null) return null;
 
   const selectionIndex = Math.min(targetInput.selectionStart, targetInput.selectionEnd);
-  const terms = getTermContexts(searchQuery);
+  const isMultiline = targetInput instanceof HTMLTextAreaElement;
+
+  let lineOffset = 0;
+  let targetQuery = searchQuery;
+
+  // Multi-line textarea elements should treat each line as the different search queries. Here we're looking for the
```

Comment:
Since the <input>s aren't normally expected to contain newline characters, could the multiline handling be done unconditionally?

Reply:
Yeah, it could be done unconditionally. I was just trying to avoid unnecessary slicing and newline characters search if we already know there should be no multiple lines at all. But if it'd be better to do it unconditionally, then I can drop the checking.

Reply:
I think it would be better, since I can't think of any advantage to not supporting multiline unconditionally.

Reply:
I've removed the condition.